<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormGestaoVendas</class>
 <widget class="QDialog" name="FormGestaoVendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDateEdit {
	border: 1px solid #bfbebd;
	border-radius: 10px;
	padding: 2px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Data de in√≠cio:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dataInicio">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Data de fim:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dataFim">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="filtrarButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border: none;
border-radius: 10px;
background-color: #6d747c;
color: #fff;
}

:hover {
background-color: #7c8289;
}</string>
       </property>
       <property name="text">
        <string>Filtrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="restaurarButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border: none;
border-radius: 10px;
background-color: #6d747c;
color: #fff;
}

:hover {
background-color: #7c8289;
}</string>
       </property>
       <property name="text">
        <string>Restaurar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Vendas</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tabelaVendas"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Produtos</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tabelaProdutosVendas"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="statusbarLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
